src/Claims.ts

```diff
@@ -150,9 +158,10 @@ export class Claims {
    * Retrieve all scopes in which claims have been made for the target identity.
    *   If the scope is an asset DID, the corresponding ticker is returned as well
    *
-   * @param args.target - identities (or identity IDs) for which to fetch claim scopes
+   * @param args.target - identitiy for which to fetch claim scopes
    *
    * @note a null scope means the identity has scopeless claims (like CDD for example)
+   * @note uses the middleware
    */
   public async getClaimScopes(args: { target: string | Identity }): Promise<ClaimScope[]> {
```

Comment:
The args should be optional and default to the current identity (this should be reflected on the comments). See `getIssuedClaims`. Also change the name to opts